app/views/coronavirus_form/session_expired.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title do %><%= t('session_expired.title') %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('session_expired.title') %>" />
+<% end %>
+
+<%= render "govuk_publishing_components/components/title", {
+  title:t("session_expired.title"),
+  margin_top: 0
+} %>
+
+<%= sanitize(t('session_expired.body_text')) %>
+
+<%= render "govuk_publishing_components/components/button", {
+  text: "Start now",
```

Comment:
Perhaps this text could 'Restart form' or something similar?

Reply:
Ah, I was given the content in the card: https://trello.com/c/4vyXO6Ww. I will ask though.